roofit/roofit/src/RooGaussian.cxx

```diff
@@ -130,3 +130,36 @@ void RooGaussian::generateEvent(Int_t code)
 
   return;
 }
+
+////////////////////////////////////////////////////////////////////////////////
+
+std::string RooGaussian::translate(RooFit::Detail::ADContextManager &ctxManager) const
+{
+   // Build a call to the stateless gaussian defined later.
+   std::string xName = ctxManager.nodeNames[x->GetName()];
+   std::string meanName = ctxManager.nodeNames[mean->GetName()];
+   std::string sigmaName = ctxManager.nodeNames[sigma->GetName()];
+   ctxManager.nodeNames[GetName()] =
+      "RooFit::Detail::EvaluateFuncs::gaussEvaluate(" + xName + ", " + meanName + ", " + sigmaName + ")";
+   return "";
+}
+
+////////////////////////////////////////////////////////////////////////////////
+
+std::string RooGaussian::buildCallToAnalyticIntegral(Int_t code, const char *rangeName,
+                                                     RooFit::Detail::ADContextManager &ctxManager) const
+{
+   std::string sigmaName = ctxManager.nodeNames[sigma->GetName()];
+   if (code == 2) {
+      std::string meanMin = std::to_string(mean.min(rangeName));
```

Comment:
Will this not cause a problem with infinity again on some build platforms?

Reply:
One solution could be to move the `valToString()` function from the unit test file to `RooNumber::toString()` ()in RooNumber.h/.cxx) and use it here.